users/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+from .models import UserProfile
+from django.utils.translation import ugettext_lazy
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = ['username', 'email']
+
+
+class CreateUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = ['id', 'username', 'email', 'password']
+        read_only_fields = ['id']
+        extra_kwargs = {'password': {'write_only': True}}
+
+    def create(self, validated_data):
+        password = validated_data['password']
+        user = UserProfile(username=validated_data['username'], email=validated_data['email'])
```

Comment:
If the user already registered with given `email` or `username` is taken by another user, this will raise an error. Can you please test it manually?